package.json

```diff
@@ -54,6 +54,7 @@
     "react-eventbrite-popup-checkout": "^1.0.5",
     "react-hook-form": "^5.7.2",
     "react-image-file-resizer": "^0.3.8",
+    "react-markdown": "^6.0.2",
```

Comment:
[note]

- https://bundlephobia.com/result?p=react-markdown@6.0.2 `32.5kB` (MINIFIED + GZIPPED)
- https://github.com/remarkjs/react-markdown/blob/main/package.json#L77-L91
  - has a whole bunch of dependencies that it pulls in